apps/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -204,10 +205,23 @@ export class PnpmOptionsConfiguration {
    */
   public readonly resolutionStrategy: ResolutionStrategy;
 
+  /**
+   * If true, then rush will block "rush install" when the lockfile has been manually edited.
+   *
+   * @remarks
+   * This causes "rush install" to fail when it is determined that the the lockfile was manually edited. This
```

Comment:
Use backticks for `rush install` since this comment gets generated as [API docs](https://rushstack.io/pages/api/rush-lib.pnpmoptionsconfiguration/)